test/distributions/test_distributions.py

```diff
@@ -3476,6 +3476,7 @@ def compute_v(x, alpha):
             ]))
 
 
+@skipIfTorchDynamo("Not a TorchDynamo suitable test")
```

Comment:
fwiw, the tests here are relatively tame and I would expect them to work with dynamo (no action needed)

Reply:
We fail only a few, but the new strategy as decided by numerous rounds of discussion is to disable whatever fails and move on. I had 3 > tests fail here so I slapped this on, and moved on. Do you want me to dig into this again?

Reply:
I think it's OK to land with some exclusions, but we should distinguish between "I don't think we should care about this test" vs "the test seems OK, but it's not really clear why things broke now, maybe we should look into it". The current message implies the former, but it seems like it's the latter

Reply:
Agreed, can change